nikola/nikola.py

```diff
@@ -455,6 +455,7 @@ def __init__(self, **config):
             'FRONT_INDEX_HEADER': '',
             'GALLERY_FOLDERS': {'galleries': 'galleries'},
             'GALLERY_SORT_BY_DATE': True,
+            'GALLERY_USE_METADATA': True,
```

Comment:
You are right. I initially added this option, thinking on conflicting processing of title. But finally, I change my mind in the same way: if there is a meta file, it is the most important source of information.

I will remove that.